### DISCUSSION TITLE
Remove Logo From Video Window

### DISCUSSION THREAD

**Discussion by Don1776** [smplayer-dev/smplayer#422] [Q&A] [answered]
Is it possible to remove the SMPlayer logo from the video window and just have black when no video is playing? Thanks.

**Comment by smplayer-dev** [comment, marked as answer]
http://old-forum.smplayer.info/viewtopic.php?f=2&t=8593

**Reply by Don1776** [reply]
That worked. Thanks!
Back story... I am running video for big screens in an arena. We want to play some video clips to the screens. Having black when nothing is playing was the final piece needed for the task. Thanks for a great program!
